common/src/main/java/com/wynntils/functions/SocialFunctions.java

```diff
@@ -7,19 +7,38 @@
 import com.wynntils.core.components.Models;
 import com.wynntils.core.functions.Function;
 import com.wynntils.core.functions.arguments.FunctionArguments;
+import java.util.List;
 
 public class SocialFunctions {
-    public static class OnlineFriendsFunction extends Function<Integer> {
+    public static class FriendsFunction extends Function<Integer> {
         @Override
         public Integer getValue(FunctionArguments arguments) {
             return Models.Friends.getFriends().size();
         }
     }
 
-    public static class OnlinePartyMembersFunction extends Function<Integer> {
+    public static class PartyMembersFunction extends Function<Integer> {
         @Override
         public Integer getValue(FunctionArguments arguments) {
-            return Models.Party.getPartyMembers().size();
+            if (arguments.getArgument("includeOffline").getBooleanValue()) {
+                return Models.Party.getPartyMembers().size();
+            } else {
+                return Models.Party.getPartyMembers().size()
+                        - Models.Party.getOfflineMembers().size();
+            }
+        }
+
+        @Override
+        public FunctionArguments.Builder getArgumentsBuilder() {
+            return new FunctionArguments.OptionalArgumentBuilder(
+                    List.of(new FunctionArguments.Argument<>("includeOffline", Boolean.class, true)));
+        }
+    }
+
+    public static class PartyLeaderFunction extends Function<String> {
+        @Override
+        public String getValue(FunctionArguments arguments) {
+            return Models.Party.getPartyLeader();
```

Comment:
What does this return when not in a party/we don't know the leader (didnt parse it yet)?

Reply:
It appears to become "??" since the returned value is null and the FunctionManager deals with that for us

Reply:
Can you check if we return null in other places? I feel like that "??" is more of error handling. I think maybe return an empty string (since we return 0 or -1 in cases where we can't figure out a number)